### DISCUSSION TITLE
Зарплата на банк

### DISCUSSION THREAD

**Discussion by LENFYS** [octothorp-team/public-dobrograd#697] [Предложения]
Предлагаю сделать так чтобы зарплата давалась не в руки, а сразу же в банк, конечно же с уведомлением, почему? Не логично что ты работаешь в ларьке (например) держишь в себе кассу, а тебе ещё и деньги приходят в руки, та даже если ты не работаешь, а просто ходишь, происходит тоже самое, думаю многие меня поддержут.

**Comment by KOwOMei** [comment]
1) Уже предлагали.
2) Тогда смысла от крайма не будет.

ПРОТИВ!!!! И ЕЩЁ РАЗ ПРОТИВ!!!!

**Reply by LENFYS** [reply]
боже, мне похуй на крайм, выглядит нрпшно, надо убрать
